pkg/services/store/auth.go

```diff
@@ -44,6 +44,12 @@ func GetUserIDString(user *user.SignedInUser) string {
 	if user == nil {
 		return ""
 	}
+	if user.IsAnonymous {
+		return "anon"
+	}
+	if user.ApiKeyID > 0 {
+		return fmt.Sprintf("key:%d", user.UserID)
+	}
```

Comment:
Are these changes intentional / desired for this PR? Didn't see any usage of `GetUserIDString` func in other backend changes

Reply:
ya -- the behavior is only seen depending on which key you use to save objects.  This is minor enough i don't mind adding it here